spec/controllers/my/collections_controller_spec.rb

```diff
@@ -18,6 +18,7 @@
         end
         @unrelated_collection = Collection.new(title: "test collection").tap do |c|
```

Comment:
Would you be OK with reducing this to:

```ruby
@unrelated_collection = Collection.create(title: "test collection") do |c|
  c.apply_depositor_metadata(FactoryGirl.create(:user).user_key)
end
```

:question: 

I'd like to see the `create` pattern in more use (vs. the `new.tap` pattern) because that makes sure the object is saved.